locust/main.py

```diff
@@ -115,6 +116,15 @@ def parse_options():
         help="Port that locust master should bind to. Only used when running with --master. Defaults to 5557. Note that Locust will also use this port + 1, so by default the master node will bind to 5557 and 5558."
     )
 
+    parser.add_option(
+        '--expect-slaves',
```

Comment:
Personally I prefer `--min-slaves` as it better communicates what the behavior is. Since you can have greater than the number of expected slaves with this implementation. See https://github.com/locustio/locust/pull/372/files#diff-6f782ed63a6a642694db58c0d5cdd932R125